src/main/java/com/celements/navigation/ListBuilder.java

```diff
@@ -29,33 +36,47 @@ public ListBuilder(String navUniqueId) {
     this.uniqueName = navUniqueId;
   }
 
+  @Override
+  @Deprecated
   public void openLevel(String mainUlCSSClasses) {
-    outStream.append("<ul" + addCssClasses(mainUlCSSClasses) + ">");
+    openLevel(new TreeSet<String>(Arrays.asList(StringUtils.split(mainUlCSSClasses, ' '))));
```

Comment:
why not LinkedHashSet here? TreeSet sorts mainUlCSSClasses lexicographically and does change the behaviour of openLevel.